src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEFieldSymbol.cs

```diff
@@ -413,7 +413,8 @@ public override Accessibility DeclaredAccessibility
                         break;
 
                     default:
-                        throw ExceptionUtilities.UnexpectedValue(_flags & FieldAttributes.FieldAccessMask);
+                        access = Accessibility.Private;
```

Comment:
See the mask at the top of the switch case. The default case only covered this one remaining value.